HACKING.md

```diff
@@ -14,6 +37,35 @@ We'd love the help!
 - We can also be found on the #snappy IRC channel on Freenode
 
 
+### Installing for development
+
+You'll need to install the development dependencies, and you'll also probably want to install it in `editable` mode so your changes actually do something:
```

Comment:
I would write something like: `so you can use your changes to the snapcraft source`..